components/02-components/02-content-group/02-news-group/news-group.scss

```diff
@@ -1,5 +1,18 @@
 /* components/02-components/02-content-group/02-news-group/news-group.scss */
 /* BEGIN news-group.scss */
 
+@media (min-width: 992px) {
+    .news3-card-laout-grey::after {
+        content: '';
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        width: 100%;
+        height: 69%;
+        background-color: #DBDEE2;
+        z-index: -1;
+    }
+}
```

Comment:
don't think this is needed, your wrapping div can have the class .lightgrey-bg, see https://utsa-asc.github.io/college-dls/components/detail/background-colors.html

Reply:
cannot get it to pick up the .lightgrey-bg and position it like in SS DLS.

There's no
.lightgrey-bg::after {
content: '';
position: absolute;
left: 0;
right: 0;
bottom: 0;
width: 100%;
height: 69%;
background-color: #e7e7e7;
z-index: -1;
}
in the site.css file

When I try adding it to the news-group.scss file, it won't pick it up.